src/components/about/About.jsx

```diff
@@ -1,161 +1,208 @@
-import { CardContainer, Center, FlexRowContainer, IconContainer, PillsContainer } from "../common/Containers.jsx";
-import { FWButton } from "../common/FWButton"
+import {
+  CardContainer,
+  Center,
+  FlexRowContainer,
+  IconContainer,
+  PillsContainer,
+  Padding,
+  SpectrumDiv,
+} from "../common/Containers.jsx";
+import { FWButton } from "../common/FWButton";
 import { Title } from "../common/Title";
 import { TitleSpan } from "../common/TitleSpan";
-import { Splide, SplideSlide } from '@splidejs/react-splide';
-import '@splidejs/react-splide/css';
-import '../../assets/card-design.png';
+import { Splide, SplideSlide } from "@splidejs/react-splide";
+import "@splidejs/react-splide/css";
+import "../../assets/card-design.png";
 import { Line } from "../common/LineDivider.jsx";
 import { Pill } from "../common/Pill.jsx";
+import { useEffect, useState } from "react";
 
 const About = () => {
+  const [width, setWidth] = useState(window.innerWidth);
 
-    return (<>
-        <Center>
+  useEffect(() => {
```

Comment:
add a clean up function in here to remove the event listener.
like: 
`return () => {
window.removeEventListener();
}`